egs/fisher_swbd/s5/local/nnet3/run_tdnn_discriminative.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
```

Comment:
Thanks. I'll try another lrate in the chain blstm smbr training, because 
tuning anything in this setup is too slow. I think I'll leave it with 
#epochs 1 and make fix in the future if necessary.

在 2016/4/14 1:20, Vimal Manohar 写道:
>
> In egs/fisher_swbd/s5/local/nnet3/run_tdnn_discriminative.sh 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59588250>:
>
> > @@ -0,0 +1,188 @@
> > +#!/bin/bash
>
> This script is fine other than possibly needing tuning for learning 
> rate and num epochs.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/a78789a804b2c83402703127757216f5e598ca29#r59588250>
>